### PR TITLE
chore(liveness): add more granular timeout messages, remove disconnec…

### PR DESCRIPTION
**Pull request by thaddmt**
…t timeout

#### Description of changes
- Added more granular timeout messaging for timeouts caused by recording delay, oval drawing delay, and failure to match oval. This is only for the error message callback function which the dev can log
- Removed the wait for disconnect timeout since the service will disconnect after 60 seconds

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/machine.test.ts [hunk 1]
```diff
@@ -695,18 +702,6 @@ describe('Liveness Machine', () => {
       );
     });
 
-    it('should reach timeout state if disconnect event never arrives', async () => {
```
Comment (esauerbo) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/machine.test.ts:698]: Is there a way to unit test for the new error messages? Is that not something we were doing previously?

Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/machine.test.ts:698]: whoops forgot to add that, will update!

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts [hunk 2]
```diff
@@ -810,7 +822,9 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
         context.componentProps!.onUserCancel?.();
       },
       callUserTimeoutCallback: (context) => {
-        const error = new Error('Client Timeout');
+        const error = new Error(
+          (context.errorMessage as string) ?? 'Client Timeout'
```
[the remark this answers is not in the document]
Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts:826]: yep it is not needed, will remove!

File: .changeset/red-points-push.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@aws-amplify/ui-react-liveness": patch
+---
+
+chore(liveness): add more granular timeout messages, remove disconnec…
```
Comment (esauerbo) [inline .changeset/red-points-push.md:5]:
```suggestion
chore(liveness): add more granular timeout messages, remove disconnect timeout
```

**Review by esauerbo — DISMISSED** [review]
LGTM

**Review by esauerbo — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]
LGTM!

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts [hunk 4]
```diff
@@ -731,15 +747,38 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
 
       // timeouts
       sendTimeoutAfterOvalDrawingDelay: actions.send(
-        { type: 'TIMEOUT' },
+        {
+          type: 'RUNTIME_ERROR',
+          data: {
+            message: 'Client failed to draw oval.',
```
Comment (reesscot) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts:753]: Do we have a constants file where we store these error message? Could we move them out to a const as a follow up item?

**Review by reesscot — APPROVED** [review]

**Pull request merged**